gradle/libs.versions.toml

```diff
@@ -3,7 +3,7 @@
 # see https://docs.gradle.org/current/samples/sample_convention_plugins.html#things_to_note
 [versions]
 checker = "3.24.0"
-ideProbe = "0.42.0"
+ideProbe = "0.41.0+30-f11e897b-SNAPSHOT"
```

Comment:
To be changed to `0.43.0` when released - if the `0.43.0` release will have the https://github.com/VirtusLab/ide-probe/pull/263 changes included